haanna/haanna.py

```diff
@@ -234,7 +234,10 @@ def get_cooling_status(self, root):
         if self.is_legacy_anna(root):
             return False # cooling not supported on legacy Anna
         locator = "appliance[type='heater_central']/logs/point_log[type='cooling_state']/period/measurement"
-        return root.find(locator).text == 'on'
+        cooling_state = root.find(locator)
+        if cooling_state:
```

Comment:
I just found out that this is not working correctly, the code must be:
`if cooling_state is not None:`
This will give the correct result

Reply:
Strange, I suspected that the cooling_state should either return an xml object if it exists or None. So if cooling_state is more pythonic.

Reply:
I wrote this on Discord before I found the solution, sorry, in Dutch:
```  
      if root.find(locator):
            return root.find(locator).text == 'on'
        return None
```
Deze code geeft niet het gewenste resultaat tov deze code:
```
        return root.find(locator).text == 'on'
```
De bovenstaande code geeft als resultaat True of False wanneer de parameter “on” of “off” is en een error wanneer de parameter niet bestaat.
De helemaal bovenstaande code geeft als resultaat True wanneer de parameter “on” is en None wanneer de parameter “off” is of niet aanwezig is.